cmd/nanotube/listen.go

```diff
@@ -92,18 +93,16 @@ func listenUDP(cfg *conf.Main, queue chan string, stop <-chan struct{}, lg *zap.
 		if err != nil {
 			lg.Error("closing the incoming connection", zap.Error(err))
 		}
-		// TODO (grzkv): Add connection counting
 	}()
 
 	buf := make([]byte, 64*1024)
 
-	residue := ""
 	in := make(chan struct{})
 	errCh := make(chan error)
 loop:
 	for {
 		go func() {
```

Comment:
You can benchmark this, I guess that having one new goroutine/datagram would be pretty expensive - and not needed, one single routine should be able to just do the Reads(). 
Not sure about the split() that we do later.

Reply:
Yes, that is not good thx.